crew

```diff
@@ -419,7 +419,7 @@ end
 
 def whatprovides (regexPat)
   fileArray = []
-  needle = regexPat.gsub(/-/,'_').gsub(/_/,'\-')
+  needle = regexPat.gsub(/-/,',').gsub(/,/,'\-')
```

Comment:
Do you mean `regexPat.gsub(/-/, '.')`? (Dots is represented to `any character` in regex)

Reply:
No, I cannot simply `.gbus(/-/, '\-')` so I need to convert the dash to a comma first.